theforeman.org/scripts/debian/configure_pbuilder.sh

```diff
@@ -4,29 +4,30 @@ set -xe
 # This script sets up the PBuilder hooks to pull in our deb repositories while the package is building
 echo "--Apt Hook Setup"
 
-for pbuilderd in /etc/pbuilder/${os}*; do
-  # Cleanup any old hooks
-  sudo rm -f ${pbuilderd}/hooks/F60addforemanrepo
+pdbuilderd=/etc/pbuilder/${os}64
```

Comment:
```suggestion
pbuilderd=/etc/pbuilder/${os}64
```

That said, looks like it's only used as `${pbuilderd}/hooks`. Perhaps `$hookdir` is a better name?